packages/cli-upload/test/upload.test.js

```diff
@@ -199,4 +200,50 @@ describe('percy upload', () => {
       '[percy] Finalized build #1: https://percy.io/test/test/123'
     ]);
   });
+
+  it('creates a new build and upload snapshots with ss token', async () => {
+    process.env.PERCY_TOKEN = 'ss_<<PERCY_TOKEN>>';
+    await upload(['./images']);
+
+    expect(logger.stderr).toEqual([]);
+    expect(logger.stdout).toEqual(jasmine.arrayContaining([
+      '[percy] Percy has started!',
+      '[percy] Uploading 3 snapshots...',
+      '[percy] Snapshot uploaded: test-1.png',
+      '[percy] Snapshot uploaded: test-2.jpg',
+      '[percy] Snapshot uploaded: test-3.jpeg',
+      '[percy] Finalized build #1: https://percy.io/test/test/123'
+    ]));
+
+    expect(api.requests['/snapshots/4567/comparisons'][0].body).toEqual({
+      data: {
+        type: 'comparisons',
+        attributes: jasmine.objectContaining({
+          'external-debug-url': null,
+          'ignore-elements-data': null
+        }),
+        relationships: {
+          tag: {
+            data: {
+              type: 'tag',
+              attributes: jasmine.objectContaining(BYOS_TAG)
+            }
+          },
+          tiles: {
+            data: jasmine.arrayContaining([{
+              type: 'tiles',
+              attributes: jasmine.objectContaining({
+                sha: jasmine.any(String)
+              })
+            }])
+          }
+        }
+      }
+    });
+  });
+
+  it('throws error for token type other than web and generic', async () => {
+    process.env.PERCY_TOKEN = 'app_invalid_token';
+    await expectAsync(upload(['./images'])).toBeRejected();
```

Comment:
```suggestion
    await expectAsync(upload(['./images'])).toBeRejected();
```

add rejected with error message expectation.

Reply:
added